executor/index_merge_reader.go

```diff
@@ -711,3 +589,72 @@ func (w *indexMergeTableScanWorker) executeTask(ctx context.Context, task *looku
 	}
 	return nil
 }
+
+type handleMerger interface {
```

Comment:
Could you add some comments for the following two structs?

Reply:
updated